### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Osoro254Alex**
### I did the following on Math magicians: components

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches the design provided.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components (All components should be class-based).
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* eslint-disable */
```
Comment (omarsalem7) [inline src/App.test.js:1]: - [ ] Kindly remove this file because it is unneeded

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Osoro254Alex 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️

- [ ] Kindly remove the files in the following image except `index.html`
>image
- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @omarsalem7, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Osoro254Alex** [comment]
@Mihndim2020 

- Requested I remove Unused files from public Directory.

@omarsalem7 

-  Checked changes made and approved.

**Pull request merged**